bindings/python/README.md

```diff
@@ -70,7 +70,8 @@ Run some tests:
 
 ```shell
 maturin develop -E test
-OPENDAL_MEMORY_TEST=on pytest -vk TestMemory
+# To run `test_write.py` and use `fs` operator
+OPENDAL_TEST=fs opendal_fs_root=/tmp pytest -vk test_write
```

Comment:
Hi, @Justin-Xiang, it's better to use `OPENDAL_FS_ROOT` to align with other parts.

Reply:
@Xuanwo Get it, should I open a PR directly or an issue?

Reply:
> should I open a PR directly

I think it's simple and can be finished in a PR directly. Thanks for the fix!